components/ILIAS/User/classes/Setup/class.ilUserSetupAgent.php

```diff
@@ -61,6 +61,9 @@ public function getUpdateObjective(Setup\Config $config = null): Setup\Objective
             ),
             new ilDatabaseUpdateStepsExecutedObjective(
                 new ilUserDB90()
+            ),
+            new ilDatabaseUpdateStepsExecutedObjective(
+                new ilUserDB10UpdateSteps()
```

Comment:
Shouldn't we use similar names here? `ilUserDB90` <-> `ilUserDB10UpdateSteps`. But of course, Stephan has to decide upon this.